docker/ftest/src/test/resources/tomcat/Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM tutum/tomcat:7.0
-
+FROM tomcat:jre8
```

Comment:
@dipak-pawar it is not urgent, but when you have 1 minute, can you update with the comments that Bartosz did so we can merge the PR?